frontend/src/pages/agency-profile-form/AgencyProfileForm.js

```diff
@@ -655,19 +752,19 @@ class AgencyProfileForm extends Component {
                   subLabel="Select Quantity if Transport Type is Available"
                   options={[
                     {
-                      title: "Pickup Truck",
-                      value: data["transportationCapacity.pickupTruck"],
-                      subkey: "pickupTruck",
+                      title: "Pick-up Truck",
+                      value: data.pickUpTruck,
+                      stateKey: "pickUpTruck",
                     },
                     {
                       title: "Van",
-                      value: data["transportationCapacity.van"],
-                      subkey: "van",
+                      value: data.van,
+                      stateKey: "van",
                     },
                     {
                       title: "Car",
-                      value: data["transportationCapacity.car"],
-                      subkey: "car",
+                      value: data.car,
+                      stateKey: "car",
                     },
                   ]}
                   stateKey="transportationCapacity"
```

Comment:
is this stateKey supposed to be here still or should it be removed now that each transportation type has its own stateKey?

Reply:
Removed :-)